tools/NodeConfigCompare/configuration/cloud_formation_template.yaml

```diff
@@ -909,6 +991,7 @@ Resources:
       Runtime: python3.8
       Role: !GetAtt 'StartUpLambdaIAMRole.Arn'
       Timeout: 60
+      ReservedConcurrentExecutions: 1
```

Comment:
Is this required?

Reply:
This was one of the warnings , so added it.
```
| WARN W92
|
| Resource: ["RedshiftConfigTestingLambda", "StartUpLambda", "LambdaFunctionGitHubCopy"]
| Line Numbers: [700, 799, 905]
|
| Lambda functions should define ReservedConcurrentExecutions to reserve simultaneous executions
```